toolchain/sem_ir/impl.h

```diff
@@ -45,13 +45,19 @@ struct Impl : public EntityWithParamsBase,
     out << "{self: " << self_id << ", constraint: " << constraint_id << "}";
   }
 
+  // Determines whether the definition of this impl has begun. This is false
```

Comment:
Would it make sense to move this onto `EntityWithParamsBase`?

Reply:
Sure.